plugins/modules/packaging/os/pacman.py

```diff
@@ -392,17 +392,15 @@ def check_packages(module, pacman_path, packages, state):
 def expand_package_groups(module, pacman_path, pkgs):
     expanded = []
 
+    __, stdout, __ = module.run_command([pacman_path, "--sync", "--groups", "--quiet"], check_rc=True)
+    available_groups = stdout.splitlines()
+
     for pkg in pkgs:
         if pkg:  # avoid empty strings
```

Comment:
this one is redundant, since https://github.com/ansible-collections/community.general/blob/875de085f7be065011f6724561c831c12e191b72/plugins/modules/packaging/os/pacman.py#L449 already checks for it